content/contents+de.lr

```diff
@@ -58,49 +58,49 @@ image:
 ----
 image_alignment: right
 #### text-block ####
-title: ### Native Python Werkzeuge...
+title: ### Native Python-Werkzeuge...
 ----
-text: Python hat sich als eine fähige Sprache bewährt – zugänglich für Neulinge, aber trotzdem mächtig in den Händen von Experten. Warum sollte Python nicht überall dort verwendet werden, wo du einem Computer sagen willst, was er zu tun hat? Sollten deine Entwicklerwerkzeuge nicht alle Fähigkeiten der Programmiersprache Python ausnutzen können und nicht nur jene, die gut zu den C-Bindungs passen?
+text: Python hat sich als eine fähige Sprache bewährt – zugänglich für Neulinge, aber trotzdem mächtig in den Händen von Experten. Warum sollte Python nicht überall dort verwendet werden, wo du einem Computer sagen willst, was er zu tun hat? Sollten deine Programme nicht den vollen Umfang von Python ausnutzen können, anstatt nur die Teile, die gut zu einer C-Schnittstelle passen?
 ----
 image: python.png
 ----
 image_alignment: right
 #### text-block ####
 title: ### ...auf Handy & Desktop...
 ----
-text: Interaktion mit Computern findet heutzutage nicht mehr in einer 80x25 Konsole statt. Es passiert auf **Handys, Tablets und Desktop-Computern**, mit ansprechenden grafischen Benutzeroberflächen. Solltest du nicht in der Lage sein, Python auf all diesen Plattformen zu nutzen, um ihre **einzigartigen Vorzüge auszuschöpfen**?
+text: Heutige Computer sind nicht mehr nur eine 80x25-Konsole, sondern **Handys, Tablets und Desktop-Computer** mit ansprechenden grafischen Benutzeroberflächen. Solltest du nicht in der Lage sein, Python auf all diesen Plattformen zu nutzen, um ihre **einzigartigen Vorzüge auszuschöpfen**?
 ----
 image: cross-platform.png
 ----
 image_alignment: left
 #### text-block ####
 title: ### ...die sich nativ verhalten.
 ----
-text: Endnutzer sollten sich nicht darum kümmern müssen, noch sollten sie erkennen, in welcher Programmiersprache ihre App geschrieben wurde. Es fängt damit an, dass die Programme wie native Programme aussehen und sich auch so verhalten. Natives Aussehen, natives Verhalten und Verteilung durch die selben Kanäle wie native Apps. Warum sollten deine Python-Werkzeuge nicht genauso gut funktionieren wie ein natives Werkzeug?
+text: Endnutzer sollten sich nicht darum kümmern müssen, noch sollten sie erkennen, in welcher Programmiersprache ihre Apps geschrieben wurden. Sie sollten wie native Apps aussehen, sich auch so verhalten und auf demselben Weg wie native Apps erhältlich sein. Warum sollten deine Python-Programme nicht genauso gut funktionieren wie native Programme?
```

Comment:
> Endnutzer sollten sich ~nicht~ **weder** darum kümmern müssen, noch

Reply:
> **Sie** sollten wie native Apps aussehen [...]

Die Endnutzer sollten wie Apps aussehen?!

Reply:
> > Endnutzer sollten sich ~~nicht~~ **weder** darum kümmern müssen, noch

Ehrlich gesagt hatte ich überlegt ob man den Nebensatz vielleicht ganz weglassen sollte. Stattdessen könnte man den Teil umschreiben zu "Endnutzern sollte nicht auffallen, in welcher Programmiersprache [...]" oder so ähnlich - dann wäre der Satz etwas leichter zu lesen.

> Die Endnutzer sollten wie Apps aussehen?!

Ja. /s

Stimmt, der Teil ist etwas mehrdeutig formuliert - gemeint ist natürlich, dass die Apps von den Benutzern nativ aussehen sollen 🙂 Werd ich umschreiben.

Reply:
> Ehrlich gesagt hatte ich überlegt ob man den Nebensatz vielleicht ganz weglassen sollte

Stimmt. Die Frage ist, welchen Teil behält man? Sich nicht kümmern müssen oder es nicht erkennen? Ich tendiere zu Letzterem, da z.B. Electron-basierte Apps doch sehr deutlich erkennbar sind.